src/rust-2024/rpit-lifetime-capture.md

```diff
@@ -1,12 +1,313 @@
-# RPIT lifetime capture
+# RPIT lifetime capture rules
 
 🚧 The 2024 Edition has not yet been released and hence this section is still "under construction".
 
-This feature is partially implemented, and not yet ready for testing.
-More information may be found in the tracking issue at <https://github.com/rust-lang/rust/issues/117587>.
+This chapter describes changes related to the **Lifetime Capture Rules 2024** introduced in [RFC 3498], including opaque type *precise capturing* introduced in [RFC 3617].
+
+[RFC 3498]: https://github.com/rust-lang/rfcs/pull/3498
+[RFC 3617]: https://github.com/rust-lang/rfcs/pull/3617
 
 ## Summary
 
+- In all editions, `use<..>` bounds allow for capturing only the lifetime parameters needed in RPIT opaque types.
```

Comment:
"needed" is awkward here.

Reply:
I feel like this puts the cart before the horse. I'd probably describe the implcit behavior first, since that's why users are here in the first place, and then describe `use<>` as the convenient fix to all the capturing woes.

Reply:
I've remove the word and concept "needed" here, rearranged these bullet points, and removed one of the bullet points about `use<..>`.

Reply:
The other `use<..>`-related bullet point has now been removed also.